flake.nix

```diff
@@ -281,6 +281,7 @@
           toml2json
           llvmPackages.lldb # This ensures the right lldb is in the environment for running rust-lldb
           wasm-bindgen-cli
+          jq
```

Comment:
Why is this necessary now?

Reply:
this is not necessary but better to have around in the shell